src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
@jcouv This is 2nd time this has happened to me. It seems to happen when I use the built in edit feature on GitHub. (So maybe I'm missed setting some where.)